Healthy/Classes/Modules/Splash/DemoViewController.swift

```diff
@@ -0,0 +1,225 @@
+//
+//  OnboardingViewController.swift
+//  Healthy
+//
+//  Created by dodo on 15/05/2023.
+//
+
+
+import UIKit
+
+class DemoViewController: UIPageViewController {
+
+    var pages = [UIViewController]()
+    
+    let skipButton = UIButton()
+    let nextButton = UIButton()
```

Comment:
I have some suggestions here, you use a programmatic style (not XIB || Storyboard ) so a performance-wise we can do it better with lazy and closures and don't need to make a function for `Style` 
make button with lazy var will keep it uninitialized until it loaded and use her action
```suggestion
    private lazy var skipButton : UIButton = {
        let btn = UIButton()
        btn.translatesAutoresizingMaskIntoConstraints = false
        btn.setTitleColor(.systemBlue, for: .normal)
        btn.setTitle("Skip", for: .normal)
        btn.addTarget(self, action: #selector(skipTapped(_:)), for: .primaryActionTriggered)
        return btn
    }()
    private lazy var nextButton : UIButton = {
        let btn = UIButton()
        btn.translatesAutoresizingMaskIntoConstraints = false
        btn.setTitleColor(.systemBlue, for: .normal)
        btn.setTitle("Next", for: .normal)
        btn.addTarget(self, action: #selector(nextTapped(_:)), for: .primaryActionTriggered)
        return btn
    }()
```

Reply:
- Note: Please use `button intead of `btn`. 
- Add TODO to update the localized strings

Reply:
📝📍